packages/Webkul/Core/src/Helpers/Laravel5Helper.php

```diff
@@ -5,82 +5,60 @@
 use Webkul\Checkout\Models\CartItem;
-use Illuminate\Support\Facades\Event;
 use Webkul\Product\Models\Product;
+use Webkul\Attribute\Models\Attribute;
 use Webkul\Checkout\Models\CartAddress;
 use Webkul\Product\Models\ProductInventory;
 use Webkul\Customer\Models\CustomerAddress;
+use Webkul\Attribute\Models\AttributeOption;
 use Webkul\Product\Models\ProductAttributeValue;
 use Webkul\Product\Models\ProductDownloadableLink;
 use Webkul\Product\Models\ProductDownloadableLinkTranslation;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Event;
 
+/**
+ * Class Laravel5Helper
+ *
+ * @package Webkul\Core\Helpers
+ */
 class Laravel5Helper extends Laravel5
 {
     public const SIMPLE_PRODUCT = 1;
     public const VIRTUAL_PRODUCT = 2;
     public const DOWNLOADABLE_PRODUCT = 3;
 
     /**
-     * Returns field name of given attribute.
+     * Returns the field name of the given attribute in which a value should be saved inside
+     * the 'product_attribute_values' table. Depends on the type.
      *
      * @param string $attribute
      *
      * @return string|null
      * @part ORM
      */
-    public static function getAttributeFieldName(string $attribute): ?string
+    public static function getAttributeFieldName(string $type): ?string
     {
-        $attributes = [
-            'product_id'           => 'integer_value',
-            'sku'                  => 'text_value',
-            'name'                 => 'text_value',
-            'url_key'              => 'text_value',
-            'tax_category_id'      => 'integer_value',
-            'new'                  => 'boolean_value',
-            'featured'             => 'boolean_value',
-            'visible_individually' => 'boolean_value',
-            'status'               => 'boolean_value',
-            'short_description'    => 'text_value',
-            'description'          => 'text_value',
-            'price'                => 'float_value',
-            'cost'                 => 'float_value',
-            'special_price'        => 'float_value',
-            'special_price_from'   => 'date_value',
-            'special_price_to'     => 'date_value',
-            'meta_title'           => 'text_value',
-            'meta_keywords'        => 'text_value',
-            'meta_description'     => 'text_value',
-            'width'                => 'integer_value',
-            'height'               => 'integer_value',
-            'depth'                => 'integer_value',
-            'weight'               => 'integer_value',
-            'color'                => 'integer_value',
-            'size'                 => 'integer_value',
-            'brand'                => 'text_value',
-            'guest_checkout'       => 'boolean_value',
-        ];
 
-        if (! array_key_exists($attribute, $attributes)) {
-            return null;
-        }
+        $attributes = [];
+
+        // @todo implement json_value ?
```

Comment:
:-)
Do or remove it.

Reply:
This is a reminder for possible future implementation if JSON. But i removed the comment.